### PR TITLE
Add initial vignettes section

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

**Comment by Aariq** [comment]
I like this idea!  One thing to keep in mind is the current setup with GitHub actions does not install R or Python and therefore cannot deal with code.  If folks want to include interactive code chunks in vignettes, they'll have to render locally with `quarto render` and commit everything in `_freeze/` before pushing to GitHub.

**Comment by KristinaRiemer** [comment]
I'll add a "How to add a vignette" section to the overview with that info! Thanks @Aariq.

File: vignettes_overview.qmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Purpose
+
+The vignettes are to document technical skills and approaches that our group members learn, that are not well-documented in other places, e.g., online tutorials, blog posts.
+Creating these vignettes provide a way to share knowledge amongst and outside the group as we learn new things.
+
+## Process
+
+1.  Investigate the desired skill using existing documentation and by using the tool / approach
+2.  Summarize findings in a Quarto document and add it to this section of the Group Procedures book
```
Comment (Aariq) [inline vignettes_overview.qmd:14]: Could also just be plain .md—Quarto will render those as well.  Another (sub) step here is to add the file path to `_quarto.yml`

**Comment by Aariq** [comment]
> I'll add a "How to add a vignette" section to the overview with that info! Thanks @Aariq.

On second thought, there should probably be a generic "Editing This Book" section that all this can go in.  I can take a crack at that if you'd like, then we can just cross-reference it here.  I'm going to need to write a chapter like this for the lab handbook workshop anyways.

**Pull request merged**